test/testsymboldatabase.cpp

```diff
@@ -5611,7 +5612,7 @@ class TestSymbolDatabase : public TestFixture {
                       "static const int foo = 8;\n"
                       "}\n");
         ASSERT(db);
-        ASSERT_EQUALS(6, db->scopeList.size());
```

Comment:
I have double checked that the duplInheritedMember etc are not shown again that was supposed to be fixed when this test was added.